server/config/nl_page/nl_vertex_ai_models.yaml

```diff
@@ -0,0 +1,17 @@
+dc-all-minilm-l6-v2:
+  index: "dc-all-minilm-l6-v2-index"
+  model: "dc-all-minilm-l6-v2-model"
+
+uae-large-v1:
+  index: "uae-large-v1-index"
+  model: "uae-large-v1-model"
+
+sfr-embedding-mistral:
+  index: "sfr-embedding-mistral-index"
+  model: "sfr-embedding-mistral-model"
+
+cross-encoder-ms-marco-miniilm-l6-v2:
+  model: "cross-encoder-ms-marco-miniilm-l6-v2"
+
+cross-encoder-mxbai-rerank-base-v1:
```

Comment:
hmm, this file is actually the index?  Also, should this be in shared too?

What do you think of the following structure:

Split `vertex_ai_endpoints.yaml` into two:

1. vertex_ai_models.yaml:  Has only the models
2. vertex_ai_indexes.yaml:  Has the names in this file, with a reference to the model (in vertex_ai_models.yaml) and endpoint info.

NL Server only reads #1 fully.

This way, we don't need `sfr-embedding-mistral` separate from `sfr-embedding-mistral-index` and `sfr-embedding-mistral-model`.

Reply:
oh I like that, done